RecoBTag/ONNXRuntime/test/test_particle_net_cfg.py

```diff
@@ -24,6 +28,23 @@
 ## Maximal Number of Events
 process.maxEvents = cms.untracked.PSet(input=cms.untracked.int32(options.maxEvents))
 
+process.load("HeterogeneousCore.SonicTriton.TritonService_cff")
+process.TritonService.verbose = True
+# fallback server
+process.TritonService.fallback.enable = True
+process.TritonService.fallback.verbose = True
+process.TritonService.fallback.useDocker = False
+process.TritonService.fallback.useGPU = False
```

Comment:
unnecessary, in `TritonService_cff` there's a check for GPU availability before configuring the fallback server

Reply:
Deleted